typedb/simulation.gql

```diff
@@ -42,12 +44,17 @@ person sub entity,
     owns birth-date,
     owns gender,
     owns address,
+    owns marital-status,
     plays birth-place:child,
     plays residentship:resident,
+    plays nationality:national,
```

Comment:
`national` is a role for a `person`? are you sure? Dictionary says the meaning of `national` is:

> relating to or characteristic of a nation; common to a whole nation.

Doesn't sound like something a `person` would be, no?

Reply:
That's the meaning when used as an adjective, used as a noun we have 

> a citizen of a particular country

To conform we'll also use `nation` rather than `state` as the other role in `nationality`.